Index: ./test_files/enum/enum.js
===================================================================
--- ./test_files/enum/enum.js	golden
+++ ./test_files/enum/enum.js	tsickle with transformer
@@ -12,10 +12,10 @@
 // Verify that the resulting TypeScript still allows you to index into the enum with all the various
 // ways allowed of enums.
 let /** @type {EnumTest1} */ enumTestValue = EnumTest1.XYZ;
 let /** @type {EnumTest1} */ enumTestValue2 = EnumTest1['XYZ'];
-let /** @type {string} */ enumNumIndex = EnumTest1[((null))];
-let /** @type {number} */ enumStrIndex = EnumTest1[((null))];
+let /** @type {string} */ enumNumIndex = EnumTest1[/** @type {number} */ ((null))];
+let /** @type {number} */ enumStrIndex = EnumTest1[/** @type {string} */ ((null))];
 /**
  * @param {EnumTest1} val
  * @return {void}
  */
@@ -35,8 +35,9 @@
 const ComponentIndex = {
     Scheme: 1,
     UserInfo: 2,
     Domain: 0,
+    // Be sure to exercise the code with a 0 enum value.
     UserInfo2: 2,
 };
 ComponentIndex[ComponentIndex.Scheme] = "Scheme";
 ComponentIndex[ComponentIndex.UserInfo] = "UserInfo";
@@ -46,9 +47,9 @@
 const ConstEnum = {
     EMITTED_ENUM_VALUE: 0,
 };
 exports.ConstEnum = ConstEnum;
-let /** @type {ConstEnum} */ constEnumValue = ConstEnum.EMITTED_ENUM_VALUE;
+let /** @type {ConstEnum} */ constEnumValue = 0 /* EMITTED_ENUM_VALUE */;
 /**
  * @record
  */
 function InterfaceUsingConstEnum() { }
