Index: ./test_files/enum.untyped/enum.untyped.js
===================================================================
--- ./test_files/enum.untyped/enum.untyped.js	golden
+++ ./test_files/enum.untyped/enum.untyped.js	tsickle with transformer
@@ -7,9 +7,10 @@
 EnumUntypedTest1.XYZ = 0;
 EnumUntypedTest1.PI = 3.14159;
 EnumUntypedTest1[EnumUntypedTest1.XYZ] = "XYZ";
 EnumUntypedTest1[EnumUntypedTest1.PI] = "PI";
-exports.EnumUntypedTest2 = {};
-exports.EnumUntypedTest2.XYZ = 0;
-exports.EnumUntypedTest2.PI = 3.14159;
-exports.EnumUntypedTest2[exports.EnumUntypedTest2.XYZ] = "XYZ";
-exports.EnumUntypedTest2[exports.EnumUntypedTest2.PI] = "PI";
+let EnumUntypedTest2 = {};
+exports.EnumUntypedTest2 = EnumUntypedTest2;
+EnumUntypedTest2.XYZ = 0;
+EnumUntypedTest2.PI = 3.14159;
+EnumUntypedTest2[EnumUntypedTest2.XYZ] = "XYZ";
+EnumUntypedTest2[EnumUntypedTest2.PI] = "PI";
