Index: ./test_files/use_closure_externs/use_closure_externs.js
===================================================================
--- ./test_files/use_closure_externs/use_closure_externs.js	golden
+++ ./test_files/use_closure_externs/use_closure_externs.js	tsickle with transformer
@@ -5,6 +5,6 @@
  */
 console.log('work around TS dropping consecutive comments');
 let /** @type {!NodeListOf<!HTMLParagraphElement>} */ x = document.getElementsByTagName('p');
 console.log(x);
-const /** @type {(null|!RegExpExecArray)} */ res = ((/asd/.exec('asd asd')));
+const /** @type {(null|!RegExpExecArray)} */ res = /** @type {!RegExpExecArray} */ ((/asd/.exec('asd asd')));
 console.log(res);
