Index: ./test_files/namespaced/export_namespace.js
===================================================================
--- ./test_files/namespaced/export_namespace.js	golden
+++ ./test_files/namespaced/export_namespace.js	tsickle with transformer
@@ -17,5 +17,5 @@
     function Interface() { }
     typeNamespace.Interface = Interface;
     function Interface_tsickle_Closure_declarations() {
     }
-})(typeNamespace = exports.typeNamespace || (exports.typeNamespace = {}));
+})(typeNamespace = typeNamespace || (typeNamespace = {}));
